hypershift-operator/controllers/scheduler/placeholders.go

```diff
@@ -66,12 +66,12 @@ func (r *PlaceholderScheduler) SetupWithManager(ctx context.Context, mgr ctrl.Ma
 			}
 			return &deployments, nil
 		},
-		listNodes: func(ctx context.Context, opts ...client.ListOption) (*corev1.NodeList, error) {
-			nodes := corev1.NodeList{}
-			if err := mgr.GetClient().List(ctx, &nodes, opts...); err != nil {
-				return nil, fmt.Errorf("could not list nodes: %w", err)
+		listConfigMaps: func(ctx context.Context, opts ...client.ListOption) (*corev1.ConfigMapList, error) {
+			configMaps := corev1.ConfigMapList{}
+			if err := mgr.GetClient().List(ctx, &configMaps, opts...); err != nil {
```

Comment:
should listConfigMaps include the namespace filter instead of passing it through in all invokations?